### DISCUSSION TITLE
关于哪吒探针的小问题

### DISCUSSION THREAD

**Discussion by openMJJ** [nezhahq/nezhahq.github.io#43] [Q&A] [answered]
有个关于哪吒探针的小问题。
报警功能有个循环流量统计功能，这个功能的“下次检查”时间频率是否可以修改。

https://tc.openface.cc/file/2bf2107c5e01076336292.png
![3c73b9d767f82ea393271daad1df44c6_2bf2107c5e01076336292](https://github.com/nezhahq/nezhahq.github.io/assets/125738740/78472170-e069-439d-8d03-a52cbdb06e1d)

**Comment by naiba** [comment, marked as answer]
这个频率是动态的，越接近阈值间隔越短

**Comment by openMJJ** [comment]
谢谢大佬了。不过还是想自己控制。
